### PR TITLE
SpatialAudioStyle implementation in flutter and iOS

### PR DESCRIPTION
**Pull request by Bindu-Maharudrappa**

### CODE REVIEW

File: test_app/lib/screens/join_screen.dart [hunk 1]
```diff
@@ -153,62 +154,88 @@ class _JoinConferenceContentState extends State<JoinConferenceContent> {
...
+                            }
+                          }),
+                      DropdownButton<String>(
+                        focusColor: Colors.white,
+                        value: switchDolbyVoice? spatialAudioStyleText : null,
+                        style: const TextStyle(color: Colors.white),
+                        iconEnabledColor: Colors.black,
+                        items: <String>[
+                          'Spatial Audio with Individual',
+                          'Spatial Audio with Shared Scene',
```
Comment (graduad) [inline test_app/lib/screens/join_screen.dart:193]: I thought the spatial audio style enum had 3 values.

File: test_app/lib/screens/join_screen.dart [hunk 2]
```diff
@@ -294,15 +321,22 @@ class _JoinConferenceContentState extends State<JoinConferenceContent> {
     var params = ConferenceCreateParameters();
     params.dolbyVoice = switchDolbyVoice;
     params.liveRecording = true;
-    var createOptions = ConferenceCreateOption(conferenceName, params, 0);
+    var createOptions = ConferenceCreateOption(
+        conferenceName, params, 0, SpatialAudioStyle.disabled);
+    if (spatialAudio == true && spatialAudioStyleText == 'Spatial Audio with Individual') {
```
Comment (graduad) [inline test_app/lib/screens/join_screen.dart:326]: Perhaps this `spatialAudioStyleText == 'Spatial Audio with Individual'` is better placed in on change, and perhaps have a constant for the string will be less error prone.

**Review by graduad — CHANGES_REQUESTED** [review]

File: test_app/lib/screens/join_screen.dart (hunk 1 above)
Comment (Bindu-Maharudrappa) [reply test_app/lib/screens/join_screen.dart:193]: Oh yes, I missed adding the last one. I will add it. thank you.

File: test_app/lib/screens/join_screen.dart [hunk 3]
```diff
@@ -294,15 +333,22 @@ class _JoinConferenceContentState extends State<JoinConferenceContent> {
...
     params.dolbyVoice = switchDolbyVoice;
     params.liveRecording = true;
-    var createOptions = ConferenceCreateOption(conferenceName, params, 0);
+    var createOptions = ConferenceCreateOption(
+        conferenceName, params, 0, SpatialAudioStyle.disabled);
+    if (spatialAudioStyleText == spatialAudioWithIndividual) {
+      createOptions.spatialAudioStyle = SpatialAudioStyle.individual;
+    } else if (spatialAudioStyleText == spatialAudioWithSharedScene) {
+      createOptions.spatialAudioStyle = SpatialAudioStyle.shared;
+    }
```
Comment (graduad) [inline test_app/lib/screens/join_screen.dart:342]: As per my previous comment, would it be possible to keep stuff like `spatialAudioStyleText == spatialAudioWithIndividual` into the `onChanged: (String? value) { }`?

Comment (Bindu-Maharudrappa) [reply test_app/lib/screens/join_screen.dart]: I updated the code to remove the check here.

**Review by graduad — DISMISSED** [review]

**Review by graduad — APPROVED** [review]

**Review by graduad — APPROVED** [review]

**Pull request merged**